src/content/h1/engine/scripting/hsc.yml

```diff
@@ -1748,15 +1749,15 @@ Functions:
         en: |-
           ```hsc
           (camera_set_dead <unit>)
-          (camera_set_dead player0)
+          (camera_set_dead (player0))
```

Comment:
Do we need to add parens around `player0`? I have some concerns that the additional parens might actually add to the script syntax nodes in some stupid way. Not sure how to test that.

Reply:
We need that -- it's not actually evaluating the static script without them.